container/README.md

```diff
@@ -0,0 +1,110 @@
+# watson-conversation-slots-intro-kube
+
+This directory allows you to deploy the watson-conversation-slots-intro application into a container running on Bluemix. 
+
+# Steps
+
+## Create the Kubernetes cluster
+
+* Follow the instructions to [Create a Kubernetes Cluster](https://github.com/IBM/container-journey-template). When you get to step 2 name your cluster Watson. 
+```
+$ bx cs cluster-create --name Watson
+```
+* If you choose a different name or already have a cluster, please substitute that name for 'Watson' in the instructions that follow.
+
+## Create the Watson Conversation Service and Bind to your Cluster
+
+Either follow the instructions to [Create a Conversation Service](https://console.ng.bluemix.net/catalog/services/conversation) or Perform the following from the Bluemix CLI:
+
+* Create the Watson Conversation service.
+
+```
+bx service create conversation free 
```

Comment:
use `\` to wrap commands.   prefix them with a `$` so its more obvious what the command is and what any relevant output is.

Reply:
I've wrapped commands >62 chars and added '$' to all